### PR TITLE
style: build section & code samples redesign

### PR DESCRIPTION
**Pull request by franky47**
Syntax highlighting theme is oneDark, feel free to switch for another (in docusaurus.config.ts).

### CODE REVIEW

File: src/components/Auth/index.tsx [hunk 1]
```diff
@@ -3,15 +3,14 @@ import Snippet from "@site/src/components/Snippet";
 export default function Auth(): JSX.Element {
   return (
     <Snippet
-      title="Plug & Play Authentication."
+      title="Seamless User Authentication."
```
Comment (peterpeterparker) [inline src/components/Auth/index.tsx:6]: The designer worked on a copy that was later changed. Let's stick to the current copy for now. - i.e. let's not change the texts.

File: src/components/Build/styles.module.scss [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+  font-size: 60px;
 }
 
 .build {
-  text-align: center;
-  max-width: 526px;
-  margin: 0 auto 2rem;
+  font-size: 20px;
+  line-height: 38px;
+  max-width: 724px;
```
Comment (peterpeterparker) [inline src/components/Build/styles.module.scss:16]: would it be worth to have a css variable for 724px - is it a value relative to a breakpoint or it's really specific?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -56,14 +56,11 @@ export default function Home(): JSX.Element {
...
-        <section className={`container ${styles.block}`}>
+        <section className={`container ${styles.block} ${styles.grid}`}>
           <Auth />
-
           <Libs />
-
-          <Serverless />
-
           <Developer />
+          <Serverless />
```
Comment (peterpeterparker) [inline src/pages/index.tsx:63]: As for the text, we can also stick to the current order of the information.

File: src/components/Build/styles.module.scss (hunk 2 above)
Comment (franky47) [reply src/components/Build/styles.module.scss:16]: It's specific to the copy, to avoid it flowing too far off the right.

Comment (peterpeterparker) [reply src/components/Build/styles.module.scss:16]: Gotcha!

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks!

**Pull request merged**